<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadersWidget</class>
 <widget class="QWidget" name="HeadersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TableWidget" name="headersTableWidget">
     <property name="styleSheet">
      <string notr="true">border:none;
selection-background-color: rgba(0, 0, 125, 100);</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>4</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="styleSheet">
      <string notr="true">
#buttonFrame {
background-image: url(:icons/backgrounds/291.jpg);
border: none;
padding: 0px;
}

#buttonFrame QPushButton {
color: #333;
border: none;
border-radius: 0px;
padding: 0px;
background: none;
}
#buttonFrame QProgressBar {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}

#buttonFrame QLCDNumber {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="7">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="readButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367895229_folder_open.png</normaloff>:/icons/icons/1367895229_folder_open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLCDNumber" name="bytesDisplay">
        <property name="numDigits">
         <number>10</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="postButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367895375_doc_empty.png</normaloff>:/icons/icons/1367895375_doc_empty.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893725_zoom.png</normaloff>:/icons/icons/1367893725_zoom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893874_playback_reload.png</normaloff>:/icons/icons/1367893874_playback_reload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="bulkDLButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/layers_1_icon&amp;48.png</normaloff>:/icons/icons/layers_1_icon&amp;48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">TableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="iconResources.qrc"/>
 </resources>
 <connections/>
</ui>
